### PR TITLE
feat: change the event size

### PR DESCRIPTION
**Pull request by efimk-lu**

### CODE REVIEW

File: src/spans/awsSpan.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ import {
   stringifyAndPrune,
   isAwsService,
   parseErrorObject,
-  omitKeys,
+  omitKeys, getEventEntitySize,
```
Comment (saartochner) [inline src/spans/awsSpan.js]: did you pretiffy?

File: src/utils.js [hunk 2]
```diff
@@ -173,6 +173,14 @@ export const isString = x =>
 
 export const MAX_ENTITY_SIZE = 1024;
 
+export const getEventEntitySize = () => {
+  if (!isNaN(parseInt(process.env['MAX_EVENT_ENTITY_SIZE']))) {
```
Comment (saartochner) [inline src/utils.js]: '''
return parseInt(process.env['MAX_EVENT_ENTITY_SIZE']) || MAX_ENTITY_SIZE
'''

File: src/utils.test.js [hunk 3]
```diff
@@ -190,6 +191,21 @@ describe('utils', () => {
...
+    process.env = { ...oldEnv };
+  });
+
+  test('getEventEntitySize NaN', () => {
+    const oldEnv = Object.assign({}, process.env);
+    process.env = { ...oldEnv, MAX_EVENT_ENTITY_SIZE: 'A 2048' };
+    expect(utils.getEventEntitySize()).toBe(MAX_ENTITY_SIZE);
+    process.env = { ...oldEnv };
+  });
+
```
Comment (saartochner) [inline src/utils.test.js]: can you also check the event+prune itself?
I.e.  give it a long event and check that it is pruned..

File: src/spans/awsSpan.js (hunk 1 above)
Comment (efimk-lu) [reply src/spans/awsSpan.js]: Mmm. It passed the tests. There was a bug in checks.sh and the circleci script. Fixing

File: src/utils.test.js (hunk 3 above)
Comment (efimk-lu) [reply src/utils.test.js]: Not following you, what's wrong with the prune tests?

Comment (saartochner) [reply src/utils.test.js]: you only checked the function `getEventEntitySize`, and not the actual change in `getFunctionSpan`

**Pull request merged**